test/run-drun/ok/stable-region-migration.drun.ok

```diff
@@ -0,0 +1,5 @@
+ingress Completed: Reply: 0x4449444c016c01b3c4b1f204680100010a00000000000000000101
+debug.print: actor0: init'ed.
+ingress Completed: Reply: 0x4449444c0000
+debug.print: actor1: checked region0.
+ingress Err: IC0503: Canister rwlgt-iiaaa-aaaaa-aaaaa-cai trapped explicitly: stable regions not enabled.
```

Comment:
@crusso This error is being trigged by `Region.new` when `actor` tries to use it.

That means that the flag isn't getting through, since that's the only way that error can arise.